webpack.config.js

```diff
@@ -160,22 +163,45 @@ module.exports = ( env ) => {
     }
   }
 
+  function getCustomIfUtils( { ifDev, ifProd, ifSite } = {} ) {
 
-  function ifProdOrSite( returnValue = true ) {
+    return {
+
```

Comment:
Not the right place to discuss it, but we should probably discuss how we want the code to be formatted to have it unified...

I was not terribly happy with new lines, but got used to put them everywhere and not aware that we have change it - maybe I did not pay attention :)

So lets just make some rules what we deffinately want and what we do not care about, because e.g. the formatter seams to be switching it one way and than the other constantly in certain files :)